web-client/iron-remote-gui/package.json

```diff
@@ -9,7 +9,7 @@
     "Zacharia Ellaham"
   ],
   "description": "Web Component providing agnostic implementation for Iron Wasm base client",
-  "version": "0.11.6",
+  "version": "0.0.0",
```

Comment:
We don’t use the version specified in this file, so I set it to 0.0.0 so that the intent is clear.